client/lib/abtest/active-tests.js

```diff
@@ -108,10 +108,10 @@ export default {
 		allowExistingUsers: true,
 	},
 	verticalSuggestedThemes: {
-		datestamp: '20191011',
+		datestamp: '20191029',
 		variations: {
-			control: 100,
-			test: 0,
+			control: 90,
+			test: 10,
```

Comment:
Overall this works for me, but is there are reason for the 90/10 split?

Reply:
@michaeldcain screenshots have known drawbacks and I'm paranoid how that will impact users.

Discussion: p1570598863005400-slack-CJEQFAAJU

This change shouldn't prevent bots from completing signup, so I'm hoping this will show fairly clear results. Once we've seen that this doesn't push signups off a cliff I might bump the test variant up to get to "conclusive" faster.